### PR TITLE
zh-translation:zh-translation/blog/2020/multiple-control-planes/index.md

### PR DESCRIPTION
**Pull request by dapengJacky**
[ ] Configuration Infrastructure
[x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Review by SataQiu — COMMENTED** [review]
/ok-to-test

**Review by rootsongjc — APPROVED** [review]
/LGTM

File: content/zh/blog/2020/multiple-control-planes/index.md [hunk 1]
```diff
@@ -9,11 +9,11 @@ keywords: [install,upgrade,revision,control plane]
 
 金丝雀部署是 Istio 的核心特性。用户依靠 Istio 的流量管理特性安全地控制新版本应用程序的推出，同时利用 Istio 丰富的遥测技术来比较金丝雀的性能。然而，当涉及到升级 Istio 时，没有一种简单的方法来监测升级，而且由于升级的就地性质，发现的问题或更改会立即影响整个网格。
 
-Istio 1.6将支持一种新的升级模式来安全部署 Istio 的新版本。在这个新模型中，代理将与它们使用的特定控制平面相关联。这允许新版本以较小的风险部署到集群—直到用户明确选择连接新版本时，才会有代理连接到新版本。这允许将工作负载逐步迁移到新的控制平面，当监控到更改时使用 Istio 遥测技术来调查任何问题，就像对工作负载使用 `VirtualService` 一样。 每个独立的控制平面被称为一个“修订”，并有一个 `istio.io/rev` 标签。
+Istio 1.6 将支持一种新的升级模式来安全部署 Istio 的新版本。在这个新模型中，代理将与它们使用的特定控制平面相关联。这允许新版本以较小的风险部署到集群—直到用户明确选择连接新版本时，才会有代理连接到新版本。这允许将工作负载逐步迁移到新的控制平面，当监控到更改时使用 Istio 遥测技术来调查任何问题，就像对工作负载使用 `VirtualService` 一样。每个独立的控制平面被称为一个“修订”，并有一个 `istio.io/rev` 标签。
 
-## 理解升级{#understanding-upgrades}
+## 认知升级{#understanding-upgrades}
```
Comment (rootsongjc) [inline content/zh/blog/2020/multiple-control-planes/index.md:14]: 这个标题的意思是解读升级过程。

Comment (dapengJacky) [reply content/zh/blog/2020/multiple-control-planes/index.md:14]: 好的

File: content/zh/blog/2020/multiple-control-planes/index.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ keywords: [install,upgrade,revision,control plane]
 
 Istio 1.6 将支持一种新的升级模式来安全部署 Istio 的新版本。在这个新模型中，代理将与它们使用的特定控制平面相关联。这允许新版本以较小的风险部署到集群—直到用户明确选择连接新版本时，才会有代理连接到新版本。这允许将工作负载逐步迁移到新的控制平面，当监控到更改时使用 Istio 遥测技术来调查任何问题，就像对工作负载使用 `VirtualService` 一样。每个独立的控制平面被称为一个“修订”，并有一个 `istio.io/rev` 标签。
 
-## 认知升级{#understanding-upgrades}
+## 解读升级过程{#understanding-upgrades}
```
Comment (rootsongjc) [inline content/zh/blog/2020/multiple-control-planes/index.md:14]:
```suggestion
## 升级过程解读{#understanding-upgrades}
```

**Pull request merged**